model/src/main/kotlin/spdx/SpdxExpression.kt

```diff
@@ -84,13 +98,18 @@ data class SpdxCompoundExpression(
 data class SpdxLicenseExceptionExpression(
         val id: String
 ) : SpdxExpression() {
+    override fun spdxLicenses(): EnumSet<SpdxLicense> = enumSetOf()
```

Comment:
Slightly shorter:

    override fun spdxLicenses() = enumSetOf<SpdxLicense>()

Reply:
Thanks, I just discovered the extension function shortly before creating the PR, so I didn't realize that the return types are not required anymore.